vendor/engines/bulk_email/app/validators/bulk_email/job_json_field_validator.rb

```diff
@@ -0,0 +1,30 @@
+module BulkEmail
+
+  class JobJsonFieldValidator < ActiveModel::Validator
```

Comment:
This was here to hopefully avoid future `rescue`s when reading these back, but if I refactor to use activerecord serialization, then yes these can probably go away.